server/graphql/v2/mutation/SupportMutations.ts

```diff
@@ -0,0 +1,60 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+
+import emailLib from '../../../lib/email';
+import { isValidEmail } from '../../../lib/utils';
+import { ValidationFailed } from '../../errors';
+
+const SUPPORT_EMAIL = 'support@opencollective.com';
+
+const SupportResponse = new GraphQLObjectType({
+  name: 'SupportResponse',
+  fields: () => ({
+    sent: {
+      type: GraphQLBoolean,
+    },
+  }),
+});
+
+const supportMessageMutation = {
+  sendMessage: {
+    type: new GraphQLNonNull(SupportResponse),
+    description: 'Send help and support message',
+    args: {
+      name: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+      email: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+      topic: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+      message: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: async (_, args): Promise<Record<string, unknown>> => {
+      if (!isValidEmail(args.email)) {
+        throw new ValidationFailed('Provide a valid email');
+      }
+
+      const subject = 'New support message';
+
+      const emailBody = `
+          Name: <strong>${args.name}</strong></br>
```

Comment:
We're probably going with a Frontend solution (I'll let @znarf comment on that) but just for the reference, we should never use template strings like that because of code injection. If you try to pass:

```es6
name = '<script>alert("Hello!")</script>'
```

You'll get a not-so-nice surprise. Of course, most email clients won't interpret the `script` tag, but you get the idea: malicious users would be able to inject what they want in this template.